### ISSUE TITLE
Show logs - no logs shown, follow logs disappear when switching tabs

### ISSUE THREAD

**Issue by Slywire** [Azure/vscode-kubernetes-tools#616]
Firstly, I love this package and I'm using it extensively. Just some minor issues on viewing the logs on VS Code for Windows.

When I right-click on a Node and select "Show Logs" it opens a logs tab in the editor, but with only the filter options and no logs displayed. 

To view a Node's logs, I can select "Follow Logs", which opens the same tab, but with logs and I can scroll through. The moment I switch tabs in my editor and come back to the logs tab, my logs vanish and again I sit with the blank window.

I'm on ms-kubernetes-tools.vscode-kubernetes-tools v1.0.4
VS Code: 
- Version: 1.38.1 (user setup)
- Commit: b37e54c98e1a74ba89e03073e5a3761284e3ffb0
- Date: 2019-09-11T13:35:15.005Z
- Electron: 4.2.10
- Chrome: 69.0.3497.128
- Node.js: 10.11.0
- V8: 6.9.427.31-electron.0
- OS: Windows_NT x64 10.0.18362

**Comment by itowlson** [comment]
cc @brendandburns

**Comment by Slywire** [comment]
Anyone else been able to replicate this issue?

**Comment by itowlson** [comment]
Sorry, I've been away and haven't had the chance to look at this.  Thanks for reporting it - I will see if I can get to it soon.

**Comment by Fredrik-C** [comment]
> Anyone else been able to replicate this issue?

Yes, I've got the same issue.
Follow logs and show logs works the first time and if you switch tabs the content disappears.
If you close the (now blanc) log tab you can not re-open any new "show logs" or "follow logs"

**Comment by derekyle** [comment]
I am getting the same issue as described by @Fredrik-C . I am able to 'show logs' or 'follow logs' for a pod one time per pod per vs code session. If I close the tab and try again for that same pod it will not work until restarting vscode completely.

**Comment by Slywire** [comment]
This issue seems to be resolved after I updated to VS Code version 1.43.2

**Comment by ahmetgeymen** [comment]
Still logs disappear when switching tabs. Using Kubernetes extension 1.2.1 and VS Code 1.51.1

```
Version: 1.51.1
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-11T01:11:34.018Z (3 wks ago)
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```
